phase1/vsphere/README.md

```diff
@@ -218,7 +218,7 @@ To access the dashboard after successful instllation of kubernetes cluster. Ther
 
 * Run ```kubectl proxy``` outside the container spawned by ```make docker-dev```
 ```
-curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.4.8/bin/linux/amd64/kubectl
+curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.5.3/bin/linux/amd64/kubectl
```

Comment:
Can you please add comment here for user to download the kubectl version that matches with kubernetes cluster they have launched?